core/app/views/refinery/admin/_tmp_search.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="search">
+  <form method='GET' action='<%= url %>'>
+    <div class="search-form">
+      <%= text_field :search, nil, :id => 'search', :type => 'search', :name => 'search', :value => local_assigns[:search],
+                     :class => 'form-control',
+                     :placeholder => t('search_input_notice', :scope => 'refinery.admin.search')
+      %>
+      <%= hidden_field :dialog, nil, :name => 'dialog', :id => nil, :value => true if from_dialog? %>
+      <%= hidden_field :callback, nil, :name => 'callback', :id => nil, :value => @callback if @callback.presence %>
+      <span id="refinery-search-btn" class="search-btn">
+        <button type="submit"><i class="icon icon-search"></i>
+          <div class="inline loader xs bump-up"></div>
+        </button>
+      </span>
+    </div><!-- /input-group -->
+  </form>
+</div>
```

Comment:
I think this comment was deleted (when the line diff was removed) so re-posting...

Note that my original PR (as well as your suggestion of an acts-as-indexed PR) breaks other search forms.

![refinery-search-wierd](https://cloud.githubusercontent.com/assets/4333395/16184382/c0fe58f4-3670-11e6-90e2-bf9d3baa23e6.png)


And my next PR would break them even more. So I've pushed what I suggest do to handle this: use a _tmp_search until all views have been moved over to use this new form. What do you think of that?

A side note: It seems odd that the html for this lives in refinerycms-acts-as-indexed. Is it just me, or does it make sense for refinerycms-acts-as-indexed to own the logic of searching and not the UI?